templates/commands/render/action_include.go

```diff
@@ -151,17 +151,25 @@ func includePath(ctx context.Context, inc *spec.IncludePath, sp *stepParams) err
 	return nil
 }
 
-// TODO(#158): add support for ignoring file name. For example, "/foo" ignores
-// a single file named foo in the root, but "foo" ignores all files named foo
-// anywhere.
 // checkIgnore checks the given path against the given patterns, if given
 // patterns is not provided, a default list of patterns is used.
-func checkIgnore(patterns []model.String, path string) (bool, error) {
+func checkIgnore(patterns []model.String, path, fromDir string) (bool, error) {
 	if len(patterns) == 0 {
 		patterns = defaultIgnorePatterns
 	}
 	for _, p := range patterns {
-		matched, err := filepath.Match(filepath.FromSlash(p.Val), path)
+		var matched bool
+		var err error
+		if filepath.Base(p.Val) == p.Val {
+			// Match file name if the pattern value is file name instead of path.
+			matched, err = filepath.Match(p.Val, filepath.Base(path))
+		} else if p.Val[0] == '/' {
+			// Match absolute path if the pattern start with a leading slash.
+			matched, err = filepath.Match(filepath.Join(fromDir, filepath.FromSlash(p.Val)), filepath.Join(fromDir, path))
```

Comment:
Check my math on this, but I think this line can be simplified to:

```
matched, err = filepath.Match(filepath.FromSlash(p.Val[1:]), path)
```

Reply:
You are right, I thought it is going to match the subdirs too but I was wrong. updating, thanks!